src/core/unit.h

```diff
@@ -444,6 +444,9 @@ typedef struct Unit {
         sd_id128_t invocation_id;
         char invocation_id_string[SD_ID128_STRING_MAX]; /* useful when logging */
 
+        /* Max. number of parallel instances allowed for a template unit */
```

Comment:
maybe should say "Refuse starting up of unit if there are already this many other units running that are instances of the same template"